setPL.sh

```diff
@@ -182,13 +181,20 @@ verifyTurbostatInstallation() {
 # script entry point
 # arguments: <PL1 value in watts> <PL2 value in watts>
 #
-if [ "$#" != "2" ]; then
+if [ "$#" == "0" ]; then
+    echo "!! Running SetPL in read-only mode !!"
+    readOnly=$TRUE
```

Comment:
Let's please refer to this as "showOnly" instead of "readOnly", which makes the intention clearer.